uk.co.compucorp.civicrm.tasksassignments/js/test/services/task_test.js

```diff
@@ -149,5 +158,45 @@ define([
         $httpBackend.flush();
       });
     });
+
+    describe("getActivityTypeIdOption()", function () {
+      beforeEach(function () {
+        $httpBackend.whenGET(/action=getoptions&debug=true&entity=Task/).respond(function () {
+          return [200, taskMock.onGetOptions.activityTypeIdFields];
+        });
+      });
+
+      it("returns promise with array of list of activity types options", function () {
+        $q.all({
```

Comment:
It should contain only the `expect`and other unavoidable statements. You should be aware of that by now. 

```javascript


var promise;

beforeEach(function() {
	promise = $q.all({
	  taskType: TaskService.getActivityTypeIdOption()
	});
});

afterEach(function() {
	$httpBackend.flush();
});

it("returns promise with array of list of activity types options", function () {
	promise.then(function(options) {
	  expect(options.taskType.obj).toBeDefined();
	  expect(options.taskType.arr).toEqual(fakeActivityTypesOptions);
	});
});
```